src/main/java/gov/nysenate/openleg/common/util/FileIOUtils.java

```diff
@@ -45,7 +45,11 @@
      * @return A collection of matching filenames
      * @throws IOException
      */
-    public static Collection<File> safeListFiles(File directory, String[] extensions, boolean recursive) throws IOException {
+    public static Collection<File> safeListFiles(File directory, String[] extensions, boolean recursive)
+            throws IOException {
+        // After shutdown is signalled, file moves shouldn't be attempted.
+        if (Thread.currentThread().isInterrupted())
```

Comment:
I recommend using opening and closing curly braces for single-line code blocks.  I know that it's not necessary to do so, but it makes for more consistent and readable code.

Reply:
I actually think single-lines are more readable, but let's make this part of our wider conversation about code style consistency!